metacat-metadata-mysql/src/test/resources/sql/prepare-test.sql

```diff
@@ -36,6 +36,20 @@ CREATE TABLE metacat.definition_metadata (
   UNIQUE KEY name (name)
 ) DEFAULT CHARSET=latin1;
 
+-- Create syntax for TABLE 'definition_metadata'
+CREATE TABLE metacat.partition_definition_metadata (
+  id bigint(20) NOT NULL AUTO_INCREMENT,
+  version bigint(20) NOT NULL,
+  created_by varchar(255) NOT NULL,
+  data longtext NOT NULL,
```

Comment:
This is a huge data type. you really need this?

Reply:
Yes, we are using this since the JSON string can be huge.